projects/lf-documentation/src/app/lf-login-documentation/lf-login-documentation.component.html

```diff
@@ -262,6 +262,13 @@
             <th>Signature</th>
             <th>Description</th>
       </tr>
+      <tr>
+        <td>authorizationRequestHandler</td>
```

Comment:
can this be moved to the bottom after the functions.. I guess you were probably thinking alphabetical, so maybe whatever makes sense
but also add that it is read-only and description should be capitalized